### PR TITLE
Setting up an initial curator-service/ui React app with CRA.

### PR DESCRIPTION
**Pull request by axmb**
Co-authored-by: allysonjp715 <allysonjp@google.com>

### CODE REVIEW

**Comment by axmb** [comment]
Made this with `create-react-app` for starters. Allyson is designing as to whether we'll use Next.js on top of React, in which case we'll probably regen this structure using the next CLI. The core React components translate mostly unchanged.

File: verification/curator-service/ui/public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (allysonjp715) [inline verification/curator-service/ui/public/manifest.json:3]: Update the above 2 lines?

**Comment by attwad** [comment]
screenshots or it did not happen ;-)

File: verification/curator-service/ui/src/App.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/App.css:24]: super-nit: it's nice to alphabetize the css rules

File: verification/curator-service/ui/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```
Comment (attwad) [inline verification/curator-service/ui/README.md:8]: should we use a default port then now that it's a different app?
3002 would be next

File: verification/curator-service/ui/src/App.css (hunk 2 above)
Comment (axmb) [reply verification/curator-service/ui/src/App.css:24]: SG, updated.

File: verification/curator-service/ui/README.md (hunk 3 above)
Comment (axmb) [reply verification/curator-service/ui/README.md]: Good catch, updated.

**Comment by axmb** [comment]
> screenshots or it did not happen ;-)

proof_for_tim

**Pull request merged**